ddtrace/contrib/langchain/patch.py

```diff
@@ -348,6 +368,14 @@ def traced_chat_model_generate(langchain, pin, func, instance, args, kwargs):
         integration.metric(span, "incr", "request.error", 1)
         raise
     finally:
+        if integration.is_pc_sampled_llmobs(span):
```

Comment:
We should rename `is_pc_sampled_llmobs()` to `is_sampled_llmobs()`, it's a bit misleading name right now since the purpose of it is to determine whether or not we submit the span to LLMObs. We do this by setting temporary tags on the span and then we'll extract them onto the LLMObs span event, so technically setting tags is a mechanism for us to submit an LLMObs span event.

Reply:
For reference, we do this already in the [OpenAI](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/contrib/openai/_endpoint_hooks.py#L230) and [Bedrock](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/contrib/botocore/services/bedrock.py#L109) integrations as well.